styles.css

```diff
@@ -27,7 +27,7 @@ div {
 }
 
 img {
-    border-radius:5px;
+    border-radius:10px;
```

Comment:
Increase border-radius 15px.
```suggestion
    border-radius:10px;
```